result/ospfv2lsa.yaml

```diff
@@ -0,0 +1,349 @@
+        router_lsas:
+          description: >-
+            One or more OSPFv2 Router-LSA - Type 1.
+          type: array
+          items:
+            $ref: '#/components/schemas/Ospfv2.RouterLsa'
+          x-field-uid: 2
+        network_lsas:
+          description: >-
+            One or more OSPFv2 Network-LSA - Type 2.
+          type: array
+          items:
+            $ref: './ospfv2lsa.yaml#/components/schemas/Ospfv2.NetworkLsa'
+          x-field-uid: 3
+        network_summary_lsas:
+          description: >-
+            One or more OSPFv2 Network summary LSA - Type 3.
+          type: array
+          items:
+            $ref: '#/components/schemas/Ospfv2.NetworkSummaryLsa'
+          x-field-uid: 4
+        summary_as_lsas:
+          description: >-
+            One or more OSPFv2 Autonomous System Boundary Router (ASBR) summary LSA - Type 4.
+          type: array
+          items:
+            $ref: './ospfv2lsa.yaml#/components/schemas/Ospfv2.SummaryAsLsa'
+          x-field-uid: 5
+        external_as_lsas:
+          description: >-
+            OSPFv2 AS-External-LSA - Type 5.
+          type: array
+          items:
+            $ref: './ospfv2lsa.yaml#/components/schemas/Ospfv2.ExternalAsLsa'
+          x-field-uid: 6
+        nssa_lsas:
+          description: >-
+            One or more OSPFv2 NSSA-LSA - Type 7.
+          type: array
+          items:
+            $ref: './ospfv2lsa.yaml#/components/schemas/Ospfv2.NssaLsa'
+          x-field-uid: 7
+        opaque_lsas:
+          description: >-
+            One or more OSPFv2 Link-Scope Opaque-LSA - Type 9.
+          type: array
+          items:
+            $ref: './ospfv2lsa.yaml#/components/schemas/Ospfv2.OpaqueLsa'
+          x-field-uid: 8
+    Ospfv2.RouterLsa:
+      description: >-
+        Contents of the router LSA.
+      type: object
+      properties:
+        header:
+          description: >-
+            Contents of the LSA header.
+          $ref: './ospfv2lsa.yaml#/components/schemas/Ospfv2.LsaHeader'
+          x-field-uid: 1
+        links:
+          description: >-
+            Links that are described within the LSA.
+          type: array
+          items:
+            $ref: '#/components/schemas/Ospfv2.Link'
+          x-field-uid: 2
+    Ospfv2.NetworkLsa:
+      description: >-
+        Contents of the Network LSA.
+      type: object
+      properties:
+        header:
+          description: >-
+            Contents of the LSA header.
+          $ref: './ospfv2lsa.yaml#/components/schemas/Ospfv2.LsaHeader'
+          x-field-uid: 1
+        network_mask:
+          description: >-
+            The IPv4 address mask for the network.
+          type: string
+          format: ipv4
+          x-field-uid: 2
+        neighbors:
+          description: >-
+            Neighbors that are described within the LSA.
+          type: array
+          items:
+            $ref: '#/components/schemas/Ospfv2.LsaNeighbor'
```

Comment:
Can't we directly have it as array of neighbor_router_ids ?

Reply:
Yes, changed the same.